connector/oidc/oidc.go

```diff
@@ -425,6 +431,28 @@ func (c *oidcConnector) createIdentity(ctx context.Context, identity connector.I
 				}
 			}
 		}
+		c.logger.Debugf("Groups::: %q \n", groups)
```

Comment:
I assume this stayed here by accident?

Reply:
This is redundant, will remove it.

Reply:
Removed in 345eba905ddb2664855e79da2acf816ad6b97eb5